detekt.yml

```diff
@@ -75,7 +75,7 @@ complexity:
     includeStaticDeclarations: false
   ComplexMethod:
     active: true
-    threshold: 10
+    threshold: 15
```

Comment:
Looks like a dirty workaround. Which method is complex that it requires this change?

Reply:
`threshold: 10` is too strict (discussed in person)